README.md

```diff
@@ -16,12 +16,12 @@ This project aims to be a full featured Telegram client with different approach
 
 ## Compiling
 
-To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Then, clone the repository using [Git](https://git-scm.com/), go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
+To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Clone the repository using [Git](https://git-scm.com/), you can use [Github Desktop](https://desktop.github.com/) if you are on Windows. Then, go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
```

Comment:
We are already on GitHub. I would just assume people know how to use git. No need for reference to external tools. Also build instructions are fully crossplatform, Windows-specific instructions are unnecessary.